### PR TITLE
Pull request as suggested in #2

### PR DESCRIPTION
**Pull request by dewey**
Updated README, added some default values and connection properties.

### CODE REVIEW

**Comment by makara** [comment]
See this https://github.com/strongloop/loopback-connector-mongodb/blob/master/lib/mongodb.js as an example. We can either use a `url` from the settings or build a `url` from:

- host
- port
- username
- password

or both, but it doesn't make sense to simply change `url` to `host`.

**Comment by dewey** [comment]
Good point, from that point of view it does indeed make more sense to just pick the `url` from our settings. I reverted the commit.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # loopback-connector-couchbase3
 without N1QL
+
+
+# Tests
+
+Tests are available in `test/` and use [mochajs](https://mochajs.org/). Install globally and run them:
```
Comment (makara) [inline README.md]: We have a makefile btw

**Comment by makara** [comment]
I don't see anything useful here. Closing.

**Pull request closed without merging**